core/app/models/spree/variant/options_presentation.rb

```diff
@@ -0,0 +1,45 @@
+class Spree::Variant::OptionsPresentation
```

Comment:
@KirillKayumov we use a nested notation for classes eg. 

```ruby
module Spree
  module Variants
    class OptionsPresenter
```

also, it would be better to put this class in `core/app/presenters`

also don't use `Spree::Variant::Something` as this messes up with code autoloader